scalafix-rules/src/main/scala/scalafix/internal/rule/DisableSyntax.scala

```diff
@@ -196,6 +196,12 @@ final class DisableSyntax(config: DisableSyntaxConfig)
               "Default args makes it hard to use methods as functions.",
               m.pos)
           }
+      case Term.ApplyInfix(_, t @ Term.Name("=="), _, _) if config.noUniversalEquality =>
+        Seq(Diagnostic(
+          "noUniversalEquality",
```

Comment:
Let's use `==` here so that the ID becomes `DisableSyntax.==`, which is consistent with `DisableSyntax.throw` and other settings